core-concepts/getters.md

```diff
@@ -115,11 +113,11 @@ export default {
 </script>
 
 <template>
-  <p>User 2: {{ getUserById(2) }}</p>
+  <p>Usuario 2: {{ getUserById(2) }}</p>
```

Comment:
I think it is not necessary to add the translation to the code examples. For didactic reasons.

Reply:
Alright, I will change other translation in code examples then